css/ampshared.css

```
@@ -357,6 +357,14 @@ amp-img:not(.i-amphtml-element)[i-amphtml-ssr] > img.i-amphtml-fill-content
   z-index: 1;
 }
 
+/* We reset the z-index of blurry image placeholders for images that are
+   rendered by the server and are not (yet) controlled by JS to speed up
+   display of the real image. This conveniently also fixes the display of
+   the image when JS fails to load. */
+amp-img.i-amphtml-ssr:not(.i-amphtml-element) > .i-amphtml-blurry-placeholder[placeholder] {
```

@sebastianbenz: We could make this a bit more generic and fix any placeholder. Would that be useful?
```suggestion
amp-img.i-amphtml-ssr:not(.i-amphtml-element) > [placeholder] {
```

---

Good point. Not sure whether it's common, but definitely enables devs to implement their own image placeholders.